.github/workflows/test.yml

```
@@ -0,0 +1,14 @@
+name: Test
+on:
+  pull_request:
+    paths:
+      - '**/*.js'
+
+jobs:
+    jest:
+        name: Run tests
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+            - run: npm install
+            - run: npm run test
```

Consistent indentation
```suggestion
jobs:
  jest:
    name: Run tests
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v4
      - run: npm install
      - run: npm run test
```